### ISSUE TITLE
关于FC-HAT

### ISSUE THREAD

**Issue by hetang-wang** [Gemaxis/GAT#1]
抱歉，打扰了，没有其他渠道能够沟通。请问你最后解决了FC-HAT git lfs文件的问题了吗？fork是不是也没有办法获取到？同研究生，期待你的回复呀~

**Comment by Gemaxis** [comment]
不好意思，没有解决，我的研究方向是 dMRI & HyperGraph，我当时是想用他的批处理超图的方法，最后自己写了dataLoader.

**Comment by hetang-wang** [comment]
> 不好意思，没有解决，我的研究方向是 dMRI & HyperGraph，我当时是想用他的批处理超图的方法，最后自己写了dataLoader.

好的，感谢你的回复~

**Comment by hetang-wang** [comment]
抱歉，又打扰你了，FC-HAT的train.py文件中的内容train_loader = Data.DataLoader(data_loader.brains_loader(idx_train), batch_size=1, shuffle=True,num_workers=0, pin_memory=True, drop_last=False)似乎表明了作者将batch设置为1了，这样网络的性能应该不太稳定。我想请教一下有没有可以参考的超图批处理方法？另外，一次对一张图进行分类，计算损失函数但并不反向传播，累积例如32张图的32个损失函数求平均进行反向传播和优化器的step的方法你觉得会是有效的吗？

**Comment by Gemaxis** [comment]
Sorry, I couldn't find any code in the hypergraph related to batching, I solved it as you said, which was also confirmed by the author of HGNN. 
Based on my experiments, implementing batching has indeed accelerated the process.

**Comment by hetang-wang** [comment]
感谢你的回复！

**Comment by Gemaxis** [comment]
不好意思，我当时的回复是错误的。我今天又看了下你的回复，我是一次对一批超图进行分类，然后将loss叠加，原理是类似你所说的，如果你还对这个有兴趣或者找到了更好的方法，欢迎加我的微信m872727085一起探讨交流。

**Comment by hetang-wang** [comment]
我暂时没再使用这种方法了，所以也没有更好的想法
